AEPIdentity/Sources/IdentityState.swift

```diff
@@ -244,7 +245,10 @@ class IdentityState {
             Log.trace(label: "\(LOG_TAG):\(#function)", "Not syncing identifiers at this time, no new identifiers or previously synced.")
             syncForIds = false
         } else {
-            generateAndPersistECID()
+            if (identityProperties.ecid == nil) {
```

Comment:
I see some new code changes, are these paths covered by tests?

Reply:
The nil check is the same as done in `generateAndPersistECID()` so the messages are only logged when the ECID is actually generated.